torchtune/models/llama2/_component_builders.py

```diff
@@ -103,7 +94,7 @@ def llama2(
         v_proj=nn.Linear(embed_dim, num_kv_heads * head_dim, bias=False),
         output_proj=nn.Linear(embed_dim, embed_dim, bias=False),
         pos_embeddings=rope,
-        kv_cache=kv_cache,
+        kv_cache=None,
```

Comment:
Is this a useless arg now? Seems if we are building KV caches at the decoder level it will always be none?

Reply:
Hmm, yeh its default to None so I don't think it needs to exist in this call.